Tetrahedral_remeshing/doc/Tetrahedral_remeshing/PackageDescription.txt

```diff
@@ -46,13 +46,17 @@ while preserving the input geometric curve and surface features.
 - `RemeshingTriangulationTraits_3`
 - `RemeshingVertexBase_3`
 - `RemeshingCellBase_3`
+- `RemeshingSizingField_3`
 
 \cgalCRPSection{Classes}
 
 - `CGAL::Tetrahedral_remeshing::Remeshing_vertex_base_3`
 - `CGAL::Tetrahedral_remeshing::Remeshing_cell_base_3`
 - `CGAL::Tetrahedral_remeshing::Remeshing_triangulation_3`
 
+- `CGAL::Tetrahedral_remeshing::Uniform_sizing_field`
+- `CGAL::Tetrahedral_remeshing::Adaptive_remeshing_sizing_field`
```

Comment:
`Uniform_sizing_field` should be an internal undocumented class template. It is used only internally, as a "dummy sizing field", in `convert_to_triangulation_3`.